### PR TITLE
doku updates for zypper_repository

### PR DESCRIPTION
**Pull request by computersalat**
##### ISSUE TYPE
 - Docs Pull Request

##### ANSIBLE VERSION

```
ansible 2.2.1.0
```

##### SUMMARY
Improve doc for zypper_repository

### CODE REVIEW

File: lib/ansible/modules/packaging/os/zypper_repository.py [hunk 1]
```diff
@@ -53,12 +53,11 @@
         required: false
         default: none
         description:
-            - A description of the repository
+            - A description (Name) of the repository
```
Comment (robinro) [inline lib/ansible/modules/packaging/os/zypper_repository.py:56]: Imo this makes the docs more confusing. Stating that `name` is required and the examples make the usage clear.

File: lib/ansible/modules/packaging/os/zypper_repository.py [hunk 2]
```diff
@@ -53,12 +53,11 @@
...
     disable_gpg_check:
         description:
             - Whether to disable GPG signature checking of
-              all packages. Has an effect only if state is
-              I(present).
+              all packages. Has an effect only if I(state=present).
```
Comment (robinro) [inline lib/ansible/modules/packaging/os/zypper_repository.py:60]: Ansible prefers native yaml syntax over the `a=b` notation, so I would keep the old version here.

File: lib/ansible/modules/packaging/os/zypper_repository.py [hunk 3]
```diff
@@ -88,7 +87,7 @@
     auto_import_keys:
         description:
             - Automatically import the gpg signing key of the new or changed repository.
-            - Has an effect only if state is I(present). Has no effect on existing (unchanged) repositories or in combination with I(absent).
+            - Has an effect only if I(state=present). Has no effect on existing (unchanged) repositories or in combination with I(state=absent).
```
Comment (robinro) [inline lib/ansible/modules/packaging/os/zypper_repository.py:90]: as above

**Review by robinro — CHANGES_REQUESTED** [review]
I'm aware of the alias/name confusion, but I don't think this change helps. Maybe we can extend the examples or put some `zypper lr` output there to make it more clear.
Please adjust the parts I commented on.

File: lib/ansible/modules/packaging/os/zypper_repository.py (hunk 3 above)
Comment (computersalat) [reply lib/ansible/modules/packaging/os/zypper_repository.py:90]: I just tried to follow [http://docs.ansible.com/ansible/dev_guide/developing_modules_documenting.html](URL) ... Formatting options
and I thought
```I(state=present)``` is better to understand than ```state is I(present)```

**Comment by computersalat** [comment]
Anyhow this is your 'island' and I think it is best to keep it as it is ...

**Pull request closed without merging**